Project.toml

```diff
@@ -1,19 +1,21 @@
 name = "JuliaFormatter"
 uuid = "98e50ef6-434e-11e9-1051-2b60c6c9e899"
 authors = ["Dominique Luna <dluna132@gmail.com>"]
-version = "1.0.12"
```

Comment:
oh make this 1.0.13 instead for vscode. This doesn't change any prior behaviour so we don't need to increment the version to 1.1

Reply:
I'm fine to do 1.1 but it helps the vscode folks to not bump the versions like this unless it's required.

Reply:
Oh, ok. Normally only the leftmost non-zero number signifies breaking changes, and this is a feature add, but I don't know VSCode so... One moment.